server/models/profile.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require("mongoose");
+const avaliableTime = new mongoose.Schema({
```

Comment:
Good thinking but we'll still have to make a few changes in this schema to make it fit our project use case (let's leave that for a later date though)

Reply:
Thanks Rajiv. Ya sure we can implement changes.